### DISCUSSION TITLE
v0.8.2 release?

### DISCUSSION THREAD

**Discussion by jonathantredway** [GoogleChrome/lighthouse-ci#697] [General]
Just curious when you expect the next release will go out. Are they scheduled on a timeline, or when a minimum quota of changes occur, or something else entirely? Hoping to give my team a rough estimate. Thanks!

**Comment by patrickhulce** [comment]
It's not a set schedule, just whenever significant changes build up. I can do one by EOW 👍

**Reply by jonathantredway** [reply]
Awesome! Really appreciate how responsive you've been!

**Reply by patrickhulce** [reply]
@jonathantredway published 🎉  https://github.com/GoogleChrome/lighthouse-ci/releases/tag/v0.8.2
